### ISSUE TITLE
[Feat]`placeholder` on `TextInput` rules

### ISSUE THREAD

**Issue by grgr-dkrk** [grgr-dkrk/eslint-plugin-rn-a11y#23]
**Is your feature request related to a problem? Please describe.**
On Android, `<TextInput />` does not read` accessibilityLabel` and `accessibility` if there is a` value` or `placeholder`.

The solution to this problem is not officially supported.
Therefore, we need a rule to warn that `accessibilityLabel` will not work when there is a` placeholder` prop.

This issue may be fixed in the future and should be a rule that does not enforce destructive settings.

ref: https://github.com/facebook/react-native/issues/26739

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**
